tools/run_tests/run_interop_tests.py

```diff
@@ -845,10 +848,13 @@ def cloud_to_cloud_jobspec(language,
         interop_only_options += ['--use_tls=true']
     elif transport_security == 'alts':
         interop_only_options += ['--use_tls=false', '--use_alts=true']
+    elif transport_security == 'google_default_credentials':
```

Comment:
We need to somehow handle `google_default_credentials` though. Otherwise we will `sys.exit(1)` later (line 857)

Reply:
We won't ever actually create those job specs with transport security of `google_default_credentials` though, right?

I <i>think</i> this is strangeness here is due to another issue: `_TRANSPORT_SECURITY_OPTIONS` probably doesn't actually need to contain `google_default_credentials` tests, because the google default credentials tests are generated when the interop tests runner script is invoked just with `tls`. I don't think that `TRANSPORT_SECURITY_OPTIONS` needs to be changed in this PR though BTW.

Reply:
Right. I deleted those two lines and tests seem OK.